ci/verify_quickstart/CMakeLists.txt

```diff
@@ -105,7 +105,7 @@ foreach (feature IN LISTS FEATURES)
         add_make_quickstart_target("${feature}" storage quickstart_grpc)
         continue()
     endif ()
-    if ("${feature}" STREQUAL "experimental-opentelemetry")
+    if ("${feature}" STREQUAL "opentelemetry")
```

Comment:
Seems like the code around line 126 would work fine now?

Reply:
We can't call `add_make_quickstart_target()` yet.